e2e-tests/cypress/support/appeal-submission-appeal-site-ownership/answerDidToldOtherOwnersAppeal.js

```diff
@@ -1,5 +1,5 @@
 module.exports = () => {
-  cy.get('#have-other-owners-been-told-yes').click();
+  cy.get('#have-other-owners-been-told').click();
```

Comment:
why have we removed "-yes", for a person using assistive tools, this should tell them it's a yes button

Reply:
we removed yes from id's as the convention in team 2 is to use them in the names.